en-US/firefox/tracking-protection-tour.lang

```diff
@@ -70,6 +93,10 @@ It’s easy to turn on Tracking Protection for the website you’re on by clicki
 Got it!
 
 
+;;Firefox Tracking Protection
+Firefox Tracking Protection
```

Comment:
It would be interesting to understand why this string wasn't already exposed.

Reply:
I will look into this.